File: libsignal-service/src/main/java/org/whispersystems/signalservice/internal/push/ProofRequiredResponse.java
package org.whispersystems.signalservice.internal.push;

import com.fasterxml.jackson.annotation.JsonProperty;

import java.util.List;

public class ProofRequiredResponse {

  @JsonProperty
  public String token;

  @JsonProperty
  public List<String> options;

  public ProofRequiredResponse() {}

  public String getToken() {
    return token;
  }

  public List<String> getOptions() {
    return options;
  }
}

File: lib/libsignal-service/src/main/java/org/whispersystems/signalservice/internal/push/ProofRequiredResponse.kt
/*
 * Copyright 2025 Signal Messenger, LLC
 * SPDX-License-Identifier: AGPL-3.0-only
 */

package org.whispersystems.signalservice.internal.push

data class ProofRequiredResponse(
  val token: String = "",
  val options: List<String> = emptyList()
)
